provider/inspec.rb

```diff
@@ -306,6 +306,10 @@ def markdown_format(property, indent = 1)
         description_arr += property.item_type.properties.map\
           { |prop| markdown_format(prop, indent + 1) }
         description = description_arr.join("\n\n")
+      elsif property.is_a?(Api::Type::Enum)
+        description_arr = [description, "#{'  ' * indent}Possible values:"]
+        description_arr += property.values.map { |v| "#{'  ' * indent}* #{v}" }
```

Comment:
I think you need additional indentation here: https://github.com/modular-magician/inspec-gcp/compare/auto-pr-3404-old..auto-pr-3404?short_path=b6ed1f3#diff-b6ed1f35a7be2989c6939b973a4a6245